templates/html/tpl.container.html

```diff
@@ -0,0 +1,3 @@
+<div class="ocpc-container" style="max-width: {MAX_WIDTH}; aspect-ratio: {RATIO}">
```

Comment:
missing a semicolon!

```suggestion
<div class="ocpc-container" style="max-width: {MAX_WIDTH}; aspect-ratio: {RATIO};">
```